### PR TITLE
Add minikube example in readme

### PR DESCRIPTION
**Pull request by johscheuer**
I tried to update the Readme to make it a little bit easier to start with `fission` on minikube.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,8 +90,27 @@ Or, you can use Google Container Engine's free trial to get a 3 node cluster.
...
 ### Get and Run Fission
 
-If you're using GKE, use the fission.yaml unmodified.  If you're using
-minikube, change all instances of LoadBalancer services to NodePort.
+#### local and no cloud provider
+
+If you're using minikube or no cloud provider, change all instances of LoadBalancer services to NodePort.
+
+```
+  $ curl -sL http://fission.io/fission.yaml | sed 's/LoadBalancer/NodePort/g' | kubectl create -f -
```
Comment (johscheuer) [inline README.md]: I understand your concerns and I think the best way to go is to split up the `fission.yaml` into two (actually three) files -> 
1.) Namespace, Deployments and no external services `fission.yaml`
2.) External Services  for cloud providers `fission-svcs-cloud.yaml`
3.) External Services with NodePort `fission-nodeport.yaml`

With this approach you don't have to modify two files if the tags of the container images are changed. If you are happy with this way I will modify my PR :)

Comment (soamvasani) [inline README.md]: Yes, that sounds good to me, thanks!

**Comment by johscheuer** [comment]
I updated the PR

**Comment by philips** [comment]
+1 to this!

**Comment by soamvasani** [comment]
Thanks for the effort, @johscheuer!  Merging.

**Pull request merged**